lib/const.rb

```diff
@@ -48,25 +52,44 @@
 
 CREW_CACHE_ENABLED = ENV['CREW_CACHE_ENABLED']
 
-CREW_DEST_HOME = CREW_DEST_DIR + HOME
+CREW_CONFLICTS_ONLY_ADVISORY = ENV['CREW_CONFLICTS_ONLY_ADVISORY']
+
+CREW_FHS_NONCOMPLIANCE_ONLY_ADVISORY = ENV['CREW_FHS_NONCOMPLIANCE_ONLY_ADVISORY']
 
 # Set CREW_NPROC from environment variable or `nproc`
-if ENV["CREW_NPROC"].to_s.empty?
+if ENV['CREW_NPROC'].to_s.empty?
   CREW_NPROC = `nproc`.strip
 else
-  CREW_NPROC = ENV["CREW_NPROC"]
+  CREW_NPROC = ENV['CREW_NPROC']
 end
 
 # Set CREW_NOT_COMPRESS from environment variable
-CREW_NOT_COMPRESS = ENV["CREW_NOT_COMPRESS"]
+CREW_NOT_COMPRESS = ENV['CREW_NOT_COMPRESS']
 
 # Set CREW_NOT_STRIP from environment variable
-CREW_NOT_STRIP = ENV["CREW_NOT_STRIP"]
+CREW_NOT_STRIP = ENV['CREW_NOT_STRIP']
+
+CREW_SHRINK_ARCHIVE = ENV['CREW_SHRINK_ARCHIVE']
+
+# Set testing constants from environment variables
+crew_testing_repo = ENV['CREW_TESTING_REPO']
+crew_testing_branch = ENV['CREW_TESTING_BRANCH']
+crew_testing = ENV['CREW_TESTING']
+crew_testing = '0' if crew_testing_repo.nil? || crew_testing_repo.empty?
+crew_testing = '0' if crew_testing_branch.nil? || crew_testing_branch.empty?
+CREW_TESTING = crew_testing
```

Comment:
Does we need both variables?

Reply:
let's merge this and then we can back out a duplicate if necessary....